pkg/managed/utils_test.go

```diff
@@ -217,3 +217,51 @@ func TestGetFilteredConnections(t *testing.T) {
 		assert.Contains(expectedClusterNames, name, "connection %s not expected", name)
 	}
 }
+
```

Comment:
standard "given/when/then" type test comments needed. Also, q - this the naming convention we use for private functions?